src/storage/src/hummock/cache.rs

```diff
@@ -933,9 +934,12 @@ mod tests {
 
             cache.release(old_entry);
             assert!(!(*old_entry).is_in_cache());
-            assert!(!(*old_entry).is_in_lru());
             assert!((*new_entry).is_in_cache());
-            assert!((*new_entry).is_in_lru());
+            #[cfg(debug_assertions)]
+            {
+                assert!(!(*old_entry).is_in_lru());
+                assert!((*new_entry).is_in_lru());
+            }
```

Comment:
Use `debug_assert` here?

Reply:
`is_in_lru` only compiles when debug_assertions is enabled, while code generated from `debug_assert` is `if cfg!(debug_asserts){...}`, whose body is still compiled even when `debug_assertions` is disabled. Therefore, if we use `debug_asserts` here we will get a compile error.

Reply:
ditto